freqtrade/data/btanalysis.py

```diff
@@ -118,9 +137,56 @@ def load_backtest_stats(filename: Union[Path, str]) -> Dict[str, Any]:
     with filename.open() as file:
         data = json_load(file)
 
+    # Legacy list format does not contain metadata.
+    if isinstance(data, dict):
+        data['metadata'] = load_backtest_metadata(filename)
+
     return data
 
 
+def find_existing_backtest_stats(dirname: Union[Path, str],
+                                 run_ids: Dict[str, str]) -> Dict[str, Any]:
+    """
+    Find existing backtest stats that match specified run IDs and load them.
+    :param dirname: pathlib.Path object, or string pointing to the file.
+    :param run_ids: {strategy_name: id_string} dictionary.
+    :return: results dict.
+    """
+    # Copy so we can modify this dict without affecting parent scope.
+    run_ids = copy(run_ids)
+    dirname = Path(dirname)
+    results: Dict[str, Any] = OrderedDict([
```

Comment:
I don't think a OrderedDict is necessary.
You're not using any "enhanced" functionality of OrderedDict - and regular dics are now also ordered by default (since 3.7 iirc).